linker/shared/src/main/scala/org/scalajs/linker/checker/ErrorReporter.scala

```diff
@@ -35,6 +35,8 @@ private[checker] object ErrorReporter {
 
     private def format(arg: Any): String = {
       arg match {
+        case arg: LocalName if arg.isThis => "`this`"
+
```

Comment:
Did you mean to leave the blank line here?

Reply:
Yes, otherwise I had to shift all the arrows in the following cases, and that was not pretty.